coderd/parameters.go

```diff
@@ -59,6 +60,33 @@ func (api *API) templateVersionDynamicParameters(rw http.ResponseWriter, r *http
 		return
 	}
 
+	tf, err := api.Database.GetTemplateVersionTerraformValues(ctx, templateVersion.ID)
+	if err != nil && !xerrors.Is(err, sql.ErrNoRows) {
+		httpapi.Write(ctx, rw, http.StatusInternalServerError, codersdk.Response{
+			Message: "Failed to retrieve Terraform values for template version",
+			Detail:  err.Error(),
+		})
+		return
+	}
+
+	major, minor, err := apiversion.Parse(tf.ProvisionerdVersion)
+	// If the api version is not valid or less than 1.5, we need to use the static parameters
+	useStaticParams := err != nil || major < 1 || (major == 1 && minor < 5)
```

Comment:
they just changed this on us to 1.6 🤦‍♀️ 
```suggestion
	useStaticParams := err != nil || major < 1 || (major == 1 && minor < 6)
```

Reply:
Good catch.